tests/lib/internal-rules/internal-consistent-docs-description.js

```diff
@@ -0,0 +1,264 @@
+    RuleTester = require("../../../lib/testers/rule-tester");
+
+//------------------------------------------------------------------------------
+// Tests
+//------------------------------------------------------------------------------
+
+const ruleTester = new RuleTester();
+
+ruleTester.run("internal-consistent-docs-description", rule, {
+    valid: [
+
+        // wrong exports format: "internal-no-valid-meta" reports this already
+        [
+            "module.exports = function(context) {",
+            "    return {",
+            "        Program: function(node) {}",
+            "    };",
+            "};"
+        ].join("\n"),
+
+        // missing `meta.docs.description` property: "internal-no-valid-meta" reports this already
+        [
+            "module.exports = {",
+            "    meta: {},",
+            "    create: function(context) {",
+            "        return {};",
+            "    }",
+            "};"
+        ].join("\n"),
+        [
+            "module.exports = {",
+            "    meta: {",
+            "        docs: {",
+            "            description: 'enforce some stuff'",
+            "        }",
+            "    },",
+
+            "    create: function(context) {",
+            "        return {};",
+            "    }",
+            "};"
+        ].join("\n"),
+        [
+            "module.exports = {",
+            "    meta: {",
+            "        docs: {",
+            "            description: 'require some things'",
+            "        }",
+            "    },",
+
+            "    create: function(context) {",
+            "        return {};",
+            "    }",
+            "};"
+        ].join("\n"),
+        [
+            "module.exports = {",
+            "    meta: {",
+            "        docs: {",
+            "            description: 'disallow bad things'",
+            "        }",
+            "    },",
+
+            "    create: function(context) {",
+            "        return {};",
+            "    }",
+            "};"
+        ].join("\n")
+    ],
+    invalid: [
+        {
+            code: [
+                "module.exports = {",
+                "    meta: {",
+                "        docs: {",
+                "            description: 'do stuff'",
+                "        }",
+                "    },",
+
+                "    create: function(context) {",
+                "        return {};",
+                "    }",
+                "};"
+            ].join("\n"),
+            errors: [{
+                message: "`meta.docs.description` should start with one of the following words: enforce, require, disallow. Started with \"do\" instead.",
+                line: 4,
+                column: 26
+            }]
+        },
+        {
+            code: [
+                "module.exports = {",
+                "    meta: {",
+                "        docs: {",
+                "            description: ''",
```

Comment:
What if we have two separate messages for:
- description is empty: `'meta.docs.description' must not be empty.`
- description starts with a whitespace (should cover "all whitespace" case as well): `meta.docs.description should not start with whitespaces.`

That way we can also revert the regex changes and simplify the "firstWord" check to use `.split()` again.
What do you think?